.github/workflows/test.yml

```diff
@@ -33,16 +33,27 @@ jobs:
         run: |
           cat src/Fast-DDS/.github/workflows/colcon.meta
           colcon build \
+            --packages-up-to fastrtps --packages-skip fastrtps \
             --event-handlers=console_direct+ \
             --metas src/Fast-DDS/.github/workflows/colcon.meta \
             --mixin coverage-gcc
+          for target in RTPSStatisticsTests StatisticsDomainParticipantTests StatisticsQosTests StatisticsDomainParticipantMockTests 
+          do
+              colcon build \
+                --packages-select fastrtps \
+                --cmake-target $target \
+                --cmake-target-skip-unavailable \
+                --event-handlers=console_direct+ \
+                --metas src/Fast-DDS/.github/workflows/colcon.meta \
+                --mixin coverage-gcc
+          done
 
       - name: Run tests
         run: |
           colcon test \
             --packages-select fastrtps \
             --event-handlers=console_direct+ \
-            --ctest-args -R Statistics
+            --ctest-args -R RTPSStatisticsTests -R Statistics -E CreateParticipant
```

Comment:
Why are you excluding `CreateParticipant`?

Reply:
Ups, to prevent some dds tests that originally failed from spoiling my pull requests 🙄

Reply:
I'm going to rebase and remove it